test/export/test_export.py

```diff
@@ -3455,11 +3455,11 @@ def forward(self, x):
         self.assertExpectedInline(
             ep.graph_module.code.strip(),
             """\
-def forward(self, arg0_1, arg1_1, arg2_1):
-    cos = torch.ops.aten.cos.default(arg2_1)
+def forward(self, p_bar_linear_weight, p_bar_linear_bias, x):
```

Comment:
Just checking, retracing also results in the same naming? And .module() also preserves the naming?

Reply:
Ah almost... retracing keeps param/buffer/constant/etc. names the same, but user inputs become arg_0, arg_1, ...

I guess that's related to .module() having this input signature:
```
def forward(self, arg_0, arg_1):
    x, y, = fx_pytree.tree_flatten_spec(([arg_0, arg_1], {}), self._in_spec)
    ...
```